locales/en/translation.json

```diff
@@ -1131,5 +1128,17 @@
   "To continue you will need to add a wallet to your key.": "To continue you will need to add a wallet to your key.",
   "Add wallet": "Add wallet",
   "Key and wallet required": "Key and wallet required",
-  "To continue you will need to create a key and add a wallet.": "To continue you will need to create a key and add a wallet."
+  "To continue you will need to create a key and add a wallet.": "To continue you will need to create a key and add a wallet.",
+  "Required": "Required",
+  "CannotUseSameCodeTwice": "Cannot use the same code twice.",
+  "EnterArgArgArgToActivateYourCard": "Enter {{0}}, {{1}}, and {{2}} to activate your card.",
+  "EnterArgArgToActivateYourCard": "Enter {{0}} and {{1}} to activate your card.",
+  "EnterArgToActivateYourCard": "Enter {{0}} to activate your card.",
+  "EnterYourInformationToActivateYourCard": "Enter your information to activate your card.",
+  "Segwit": "Segwit",
+  "Testnet": "Testnet",
+  "InvalidInvitationCode": "Invalid invitation code",
+  "MustBeLessThanArgCharacters": "Must be less than {{0}} characters",
```

Comment:
Since this isn't a list, it doesn't have to be `0` I guess, it could be `max` or something, might update this in a bit